tests/integration/indexTest.js

```diff
@@ -14,50 +15,6 @@ const fixDone = done => {
     };
 };
 
-const EventEmitter = require("events");
-class TelegramBot extends EventEmitter {
```

Comment:
Well done, I had meant to move the `TelegramBot` helper to a different file for a while but never got around to doing it.